skflow/__init__.py

```diff
@@ -93,6 +93,14 @@ def _setup_training(self):
             self._trainer = TensorFlowTrainer(self._model_loss,
                 self._global_step, self.optimizer, self.learning_rate)
             self._session = tf.Session(self.tf_master)
+    
+    def _setup_summary_writer(self):
+        # Create summary to monitor loss
+        tf.scalar_summary("loss", self._model_loss)
+        # Set up a single operator to merge all the summaries
+        summary_op = tf.merge_all_summaries()
+        # Set up summary writer to a tmp directory
+        self._summary_writer = tf.train.SummaryWriter('/tmp/tensorflow_logs', graph_def=self._session.graph_def)
```

Comment:
I think this is a good direction, but few things are missing:
1) Let's have a parameter for the Estimator to specify this path (e.g. logdir parameter).
2) Let's create subfolder with for example timestamp (e.g. /tmp/tensorflow_logs/123456788) so that when you fire off tensorboard you have different models for comparison.